lib/util/git/index.spec.ts

```diff
@@ -87,6 +88,12 @@ describe(getName(), () => {
     await base.cleanup();
   });
 
+  it('has minimum git version', async () => {
+    const res = await git.gitVersion();
+    // git show-current was introduced in 2.22.0
+    expect(semver.isVersion).toBeTrue();
+    expect(semver.isGreaterThan(res, '2.21.0')).toBeTrue();
```

Comment:
```suggestion
    expect(semver.isGreaterThan(res, '2.22.0')).toBeTrue();
```

?

Reply:
2.22.0 is ok. Maybe need to add an equals ||

Reply:
yes, better use greater or equal